src/map/Map.js

```diff
@@ -514,23 +514,24 @@ export const Map = Evented.extend({
 	// padding options to fit the display to more restricted bounds.
 	// If `latlng` is already within the (optionally padded) display bounds,
 	// the map will not be panned.
-	panInside(latlng, options) {
-		options = options || {};
-
-		const paddingTL = toPoint(options.paddingTopLeft || options.padding || [0, 0]),
-		    paddingBR = toPoint(options.paddingBottomRight || options.padding || [0, 0]),
-		    pixelCenter = this.project(this.getCenter()),
-		    pixelPoint = this.project(latlng),
-		    pixelBounds = this.getPixelBounds(),
-		    paddedBounds = toBounds([pixelBounds.min.add(paddingTL), pixelBounds.max.subtract(paddingBR)]),
-		    paddedSize = paddedBounds.getSize();
-
+	panInside(latlng, options = {}) {
+		let { paddingTopLeft, paddingBottomRight, padding } = options;

```

Comment:
Everything is `const`. Because this is javascript, and when an `Object` is declared as `const`, its properties can still be changed. Which reinforces my feeling that the PR is not human-made.

Reply:
Yeah that's true